index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Amr Ahmed - Portfolio</title>
+</head>
+<body>
+    <img src="/images/top_circles.png" alt="Background image circles white">
+    <!-- <img src="/images/portfoliobg.png" alt="Background image circles white"> -->
```

Comment:
- [ ] This image is expected to be a background image, therefore, please add it as a background image in your CSS file instead of adding it as a normal image in the HTML file.